README.md

```diff
@@ -56,3 +56,5 @@ TODO:
   and then to json
 
 * Better glue for language runtimes such as python or lua
+
+* Implement zero-copy iterators on top of the dynamic
```

Comment:
Sounds good. We missed this part.
Should we order the items in the reverse order to match the priority?